rosidl_typesupport_connext_cpp/cmake/rosidl_typesupport_connext_cpp_generate_interfaces.cmake

```diff
@@ -61,11 +61,21 @@ endforeach()
 
 # If not on Windows, disable some warnings with Connext's generated code
 if(NOT WIN32)
-  set(_connext_compile_flags
-    "-Wno-tautological-compare"
-    "-Wno-return-type-c-linkage"
-    "-Wno-deprecated-register"
-  )
+  if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+    set(_connext_compile_flags
+      "-Wno-deprecated-register "
+      "-Wno-mismatched-tags "
+      "-Wno-return-type-c-linkage "
+      "-Wno-sometimes-uninitialized "
+      "-Wno-tautological-compare "
+      "-Wno-unused-variable "
```

Comment:
When I did this it would pass the flags to gcc all concatenated together. See (search for `unrecognized command line option`):

http://54.183.26.131:8080/job/ros2_batch_ci_linux/124/consoleFull
```
cc1plus: warning: unrecognized command line option "-Wno-tautological-compare-Wno-return-type-c-linkage-Wno-deprecated-register" [enabled by default]
```